### DISCUSSION TITLE
How can I get the source code of nextjs.org

### DISCUSSION THREAD

**Discussion by PengBoUESTC** [vercel/next.js#50882] [Help]
### Summary

there are some problems about the doc format, but i dont know how to create a PR...  =_=

**Comment by icyJoseph** [comment]
Unfortunately it's closed source. You can drop a Feedback message on the page, they'll typically respond quickly! Or you, if its in the learn section, you can try to open an MR in the learn repository, https://github.com/vercel/next-learn
